tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<ul><li><strong>Even when working with global addressing channels</strong>, the detuning of individual qubits can be addressed locally by ...</li><li>The <code>DetuningMap</code> consists of a set of weights on specific sites that dictate how the detuning applied by the <code>DMM</code> is distributed.</li><li>The 1/2 factor on <code>sigma_z</code> is missing here </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>